src/network/networkd-route.c

```diff
@@ -648,6 +648,12 @@ int route_configure(
                         return log_error_errno(r, "Could not append RTAX_INITRWND attribute: %m");
         }
 
+        if (route->quickack != -1) {
```

Comment:
What is the default value of `quickack`? Where is it initialized to -1?

Reply:
right fixed.